src/annotator/util/shadow-root.js

```diff
@@ -22,20 +22,10 @@ function loadStyles(shadowRoot) {
  * Create the shadow root for an annotator UI component and load the annotator
  * CSS styles into it.
  *
- * In browsers that support it, shadow DOM is used to isolate annotator UI
- * components from the host page's styles.
- *
  * @param {HTMLElement} container - Container element to render the UI into
- * @return {HTMLElement|ShadowRoot} -
- *   The element to render the UI into. This may be `container` or the shadow
- *   root.
+ * @return {ShadowRoot}
  */
 export function createShadowRoot(container) {
-  if (!container.attachShadow) {
-    stopEventPropagation(container);
-    return container;
-  }
-
   const shadowRoot = container.attachShadow({ mode: 'open' });
```

Comment:
Unrelated with this PR, but I have tried `mode: 'closed'` and the applications seems to work well too.